### ISSUE TITLE
Dataloader

### ISSUE THREAD

**Issue by HamzaGbada** [yasahiro10/arabian-calligraphy-demo#1]
1. Create a normal dataloader; expected output: `image` (cropped using bounding box), bounding box, label:

```json
{
  "images": [torch.tensor1, torch.tensor2, ..., torch.tensorN],
  "bbox": [[x,y,w,h]1, [x,y,w,h]2, ...,[x,y,w,h]N],
  "label": [label1, label2, ..., labelN]
}
```
2. Prepare the required Tests
